### ISSUE TITLE
Make imports explicit

### ISSUE THREAD

**Issue by TheoPannetier** [Giappo/jap#10]
```
checking dependencies in R code ... WARNING
  '::' or ':::' imports not declared from:
    ‘rstudioapi’ ‘usethis’
  'library' or 'require' call not declared from: ‘rJava’
  'library' or 'require' call to ‘rJava’ in package code.
    Please use :: or requireNamespace() instead.
    See section 'Suggested packages' in the 'Writing R Extensions' manual.
  Unexported object imported by a ':::' call: ‘usethis:::scoped_path_r’
    See the note in ?`:::` about the use of this operator.
```

I don't know what calls throw all these
@Giappo can you add the relevant `::` ?
Also add the fields in `Imports`

**Comment by Giappo** [comment]
I am trying to fix this but I have no idea how to deal with usethis:::scoped_path_r because if I use ::: it complains that I should use only ::, but if I use :: then the function cannot be called because it is not exported by usethis.
Very very weird

**Comment by Neves-P** [comment]
Indeed. I didn't know this, but it doesn't seem to be allowed to use an non-exported function in a package: http://r-pkgs.had.co.nz/check.html#check-checks (see section Checking R code for possible problems).

They suggest either writing a function that does the same thing, or if the license permits, incorporate that function into the package. None seem ideal

**Comment by TheoPannetier** [comment]
It seems then that they don't want people to use their internal functions.
But how about asking the maintainers of `usethis` if they could export the function?

**Comment by TheoPannetier** [comment]
I see almost all fields have been fixed, great! Besides ‘usethis:::scoped_path_r’, there is apparently still one function call that is missing the rJava:: call. I can't locate it, @Giappo do you know where it is?

**Comment by Giappo** [comment]
Try to look into jap::fix_java or, in general, try a ctrl+shift+f for rJava

**Comment by Neves-P** [comment]
I've never used this function as I don't have java problems, but assuming the intended behavior of the `require(rJava)` calls is to return boolean if the package is available or not, maybe we can replace it with a call to `requireNamespace(rJava, quietly = TRUE)` instead to avoid the warning and the same results?

**Comment by Giappo** [comment]
Give it a try!
